src/Linters/NoEmptyStatementsLinter.hack

```diff
@@ -43,21 +42,40 @@ final class NoEmptyStatementsLinter extends AutoFixingASTLinter {
         $this,
         'This statement includes an expression that has no effect',
         $stmt,
-        () ==> $this->getFixedNode($stmt),
```

Comment:
This removal doesn't change the behavior.
The autofix previously did an early return when the expression was null.
This just removes the autofix quick fix in the IDE, since it was not going to do anything anyway.